doc/source/whatsnew/v0.24.0.txt

```diff
@@ -13,10 +13,8 @@ v0.24.0 (Month XX, 2018)
 New features
 ~~~~~~~~~~~~
 - :func:`merge` now directly allows merge between objects of type ``DataFrame`` and named ``Series``, without the need to convert the ``Series`` object into a ``DataFrame`` beforehand (:issue:`21220`)
-
-
 - ``ExcelWriter`` now accepts ``mode`` as a keyword argument, enabling append to existing workbooks when using the ``openpyxl`` engine (:issue:`3441`)
-
+- ``FrozenList`` has gained the ``.union()`` and ``.difference()`` methods (:issue:`15475`, :issue:`15506`)
```

Comment:
maybe re-word this somewhat, FrozenList is not really public. Its the use of it which is public.

Reply:
Added some description on the improvements it causes for public-facing methods.